### PR TITLE
[MFTF]: User sees stock image attributes localized automation

### PR DESCRIPTION
**Pull request by engcom-Golf**
### Description (*)
Implemented MFTF test to cover scenario: "User sees stock image attributes localized"

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#197: Issue title

### Manual testing scenarios (*)
Run functional tests.

NOTE: One step not automated because functionality not provided yet -> **Assert that User views stock image details and sees attributes values in "Russian"**

### CODE REVIEW

File: AdobeStockImageAdminUi/Test/Mftf/Test/UserSeesStockImageAttributesLocalizedTest.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                <argument name="enabled" value="0"/>
+                <argument name="wysiwygEnabled" value="disabled"/>
+<!--                TODO: uncomment next lines where issue with sensitive config setting will be resolved. -->
+<!--                <argument name="apiKey" value=""/>-->
+<!--                <argument name="privateKey" value=""/>-->
+            </actionGroup>
+            <actionGroup ref="SetAdminAccountActionGroup" stepKey="setAdminInterfaceLocaleToDefaultValue">
+                <argument name="InterfaceLocaleByValue" value="en_US"/>
+            </actionGroup>
+            <amOnPage url="admin/admin/auth/logout/" stepKey="amOnLogoutPage"/>
```
Comment (hws47a) [inline AdobeStockImageAdminUi/Test/Mftf/Test/UserSeesStockImageAttributesLocalizedTest.xml]: There is a specific action group for logout action
`<actionGroup ref="logout" stepKey="logout"/>`

File: AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSection.xml [hunk 2]
```diff
@@ -23,5 +23,6 @@
         <element name="savePreview" type="block" selector="//button[@class='action-secondary']/span[contains(.,'Save Preview')]"/>
         <element name="imagePreview" type="block" selector="//div[@class='masonry-image-preview']//img"/>
         <element name="imagePreviewNavigation" type="block" selector="//div[@class='masonry-image-preview']//div[@class='action-buttons']/button[@class='action-{{type}}']" parameterized="true"/>
+        <element name="panelCloseButton" type="text" selector="//aside[@aria-labelledby='modal-title-11']/div/header/button[@class='action-close']"/>
```
Comment (hws47a) [inline AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSection.xml]: This element is not used in the merge request

**Review by hws47a — DISMISSED** [review]

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @engcom-Golf thanks for the PR! Can you please update the branch, adjust the test and resolve review comments.

Also please check if keywords can be used to test this functionality

File: AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSection.xml (hunk 2 above)
Comment (engcom-Golf) [reply AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSection.xml]: ok

**Comment by engcom-Golf** [comment]
Hi @sivaschenko thank you for your review, i'm updated PR regarding you review comments.

**Review by sivaschenko — DISMISSED** [review]

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**